### DISCUSSION TITLE
get database information

### DISCUSSION THREAD

**Discussion by yosfhaim0** [chaitu-ycr/py_canoe#51] [Ideas]
can we pull from the py_canoe database details?
like all the signal's names without knowing it,
and also which PDUs exist? Frames? Ecus? 
signal groups ... and so on?

**Comment by chaitu-ycr** [comment]
hi @yosfhaim0 

this can be acheived using `get_bus_databases_info` method.

https://chaitu-ycr.github.io/py-canoe/src_manual/#src.py_canoe.canoe.CANoe.get_bus_databases_info

this function return information like this https://github.com/chaitu-ycr/py-canoe/blob/main/src/py_canoe/core/bus.py#L110-L121
